src/cmd/flux-account-service.py

```diff
@@ -220,6 +220,8 @@ def view_bank(self, handle, watcher, msg, arg):
             handle.respond(msg, payload)
         except KeyError as exc:
             handle.respond_error(msg, 0, f"missing key in payload: {exc}")
+        except ValueError as val_err:
+            handle.respond_error(msg, 0, f"error in view-bank: {str(val_err)}")
```

Comment:
Again: I'm not sure `str(val_err)` is needed here. I do notice you use that other places, so I apologize if I"m wrong!